### PR TITLE
Creating components for input fields

### PR DESCRIPTION
**Pull request by Ekman1991**

### CODE REVIEW

**Comment by Ekman1991** [comment]
## List of input fields that have been implemented as components

- [x] First name
- [x] Last name
- [x] Email (94bf40c)
- [x] Phone (e98c2ab)
- [x] Address (b140cc9)
  - Street
  - Zip code
  - City
- [x] Password
- [ ] Repeat password
- [ ] Gender (Before implementing this merge in branch: `add-gender-to-edit-profile-page`

**Comment by bex1** [comment]
Awesome. Keep it up

File: src/app/app.module.ts [hunk 1]
```diff
@@ -69,6 +69,10 @@ import {UserManager} from './services/user-manager.service';
 import {UserProfileComponent} from './views/user/user-settings/user-profile/user-profile.component';
 import {UserProxy} from './services/proxy/user-proxy.service';
 import {UserSettingsComponent} from './views/user/user-settings/user-settings.component';
+import {TextInputComponent} from './components/form-parts/inputs/text-input/text-input.component';
```
Comment (bex1) [inline src/app/app.module.ts]: Dont forget to sort these

File: src/app/app.module.ts [hunk 2]
```diff
@@ -136,6 +140,10 @@ export class RavenErrorHandler implements ErrorHandler {
     UserDetailsFormComponent,
     UserJobsComponent,
     UserProfileComponent,
+    TextInputComponent,
```
Comment (bex1) [inline src/app/app.module.ts]: Dont forget to sort these

**Comment by buren** [comment]
Also see #375

**Comment by buren** [comment]
* Move all components from `form-parts/` => `forms/`
* Sort `app.module.ts` imports and declarations
* Sort `en.json`
* Refactor `user-profile.component` & `user-details.component`
* New components
  - `text-input.component`
  - `user-first-name-input.component`
  - `user-last-name-input.component`
  - `user-email-input.component`
  - `user-phone-input.component`
  - `user-address-input.component`
  - `user-password-input.component`
  - `user-bank-account-input.component`

**Pull request merged**